### ISSUE TITLE
OAuth Playground can no longer be used to generate indefinite refresh tokens

### ISSUE THREAD

**Issue by Bilge** [ivanvermeyen/laravel-google-drive-demo#32]
Google have updated OAuth Playground to automatically revoke refresh tokens after 24 hours.

![image](https://user-images.githubusercontent.com/470626/39916899-cc4aaace-5503-11e8-8e44-f0194174b5a1.png)

>*Note: The OAuth Playground will automatically revoke refresh tokens after 24h. You can avoid this by specifying your own application OAuth credentials using the Configuration panel.*

This message appears even when specifying the OAuth credentials.

**Comment by ivanvermeyen** [comment]
We do use our own OAuth credentials to generate the refresh token, so I think it's still ok?

https://github.com/ivanvermeyen/laravel-google-drive-demo/blob/master/README/2-getting-your-refresh-token.md

**Comment by Bilge** [comment]
Correct. I expected this message to go away when own OAuth credentials have been supplied, but it doesn't; it is always displayed.

**Comment by mendietafirpo** [comment]
I have some drive accounts configured, and they work ok, but after a few days I have to go through the process again at: https://developers.google.com/oauthplayground because it gives me a message: Daily Limit for Unauthenticated Use Exceeded. Continued use requires signup. What's up?? thank you

**Comment by ivanvermeyen** [comment]
Hi,

I haven't used Google Drive in quite a while, so I'm not up to date with the latest changes.
Maybe you can find more info in this newer package: https://github.com/masbug/flysystem-google-drive-ext/issues/33
